scripts/check_crlf.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+    Checker for line endings
+    ~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Make sure each Python does not use Windows-style file endings.
```

Comment:
`each Python file` probably?

Reply:
So someone does read the docs! Thanks, good catch.